master/usage/index.md

```diff
@@ -1,5 +1,67 @@
----
-title: Using Calico
----
-
+---
+title: Using Calico
+---
+
 This section contains information on using Calico.
+
+## Calico software components
+
+There are several Calico components that may be ran as part of your
+installation, check out the
+[Getting Started]({{site.baseurl}}/{{page.version}}/getting-started)
+section for the components that are needed for your specific orchestrator.
+Here are the common components with links to relevant sections for configuring
+and operating those components.
+
+- [Calico node container]({{site.baseurl}}/{{page.version}}/reference/node/configuration)
```

Comment:
Is a user going to know _when_ he/she wants to click on one of these?  See my comment above about why they're coming to this page.

This is probably a good compilation of various configuration references, but maybe it needs its own landing page with more detail on each of these bullets about why you might want to click each one.